rollbar-spring-webmvc/src/main/java/com/rollbar/spring/webmvc/RollbarExceptionResolver.java

```diff
@@ -0,0 +1,34 @@
+package com.rollbar.spring.webmvc;
+
+import com.rollbar.notifier.Rollbar;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.servlet.HandlerExceptionResolver;
+import org.springframework.web.servlet.ModelAndView;
+
+@Order(Ordered.HIGHEST_PRECEDENCE)
+@ControllerAdvice
+public class RollbarExceptionResolver implements HandlerExceptionResolver {
```

Comment:
honestly i don't have a strong opinion here.. but I can change it since that's how some of the other classes are named

Reply:
@mrunalk I had a chance to look at the other classes.. Since I am implementing HandlerExceptionResolver I think it would make sense to rename it to RollbarHandlerExceptionResolver, otherwise I think we would need to pre-fix all the classes in the SDK with RollbarSpring including RollbarWebApplicationInitializer. Lmk if what you think.

Reply:
I went ahead and renamed it to RollbarHandlerExceptionResolver.java, lmk what you think